python/ray/rllib/models/catalog.py

```diff
@@ -51,14 +51,15 @@ class ModelCatalog(object):
         >>> prep = ModelCatalog.get_preprocessor(env)
         >>> observation = prep.transform(raw_observation)
 
-        >>> dist_cls, dist_dim = ModelCatalog.get_action_dist(env.action_space)
-        >>> model = ModelCatalog.get_model(inputs, dist_dim)
+        >>> dist_cls, dist_dim = ModelCatalog.get_action_dist(
```

Comment:
why these changes (making config/option mandatory)?

Reply:
This prevents you from forgetting to specify them. We should have made it mandatory in the first place.